src/xsar/raster_readers.py

```diff
@@ -67,7 +67,6 @@ def ecmwf_0100_1h(fname):
             '10V': 'V10'
         }
     )
-    ecmwf_ds['WSPD'] = np.sqrt(ecmwf_ds.U10**2 + ecmwf_ds.V10**2)
```

Comment:
@vincelhx , note that I have *removed* `ecmwf_ds['WSPD']`. If you use it, you will have to recompute it